src/sentry/integrations/msteams/webhook.py

```diff
@@ -200,6 +202,55 @@ def handle_member_removed(self, request):
         integration.delete()
         return self.respond(status=204)
 
+    def make_action_data(self, identity, data):
+        action_data = {}
+        if data["actionType"] == "unresolve" or data["actionType"] == "unignore":
+            action_data = {"status": "unresolved"}
+            analytics.record("integrations.msteams.unresolve", actor_id=identity.user_id)
```

Comment:
Hmmm, for `analyrics.record` to work, you need to create an `analytics.Event`, like in [this](https://github.com/getsentry/sentry/pull/19890/files) PR.

Reply:
That's weird, they don't do that for the [similar functionality in Slack](https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/slack/action_endpoint.py).

Reply:
Me thinks there is: https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/slack/analytics.py